common/dllserver/thorplugin.cpp

```diff
@@ -166,7 +165,6 @@ static bool getResourceFromMappedFile(const char * filename, const byte * start_
         }
     }
 
-    DBGLOG("Failed to extract resource %s: Does not include a matching entry", filename);
```

Comment:
I'm debating whether these should be present or not.  It probably doesn't matter much either way.

Reply:
They are presently appearing - and confusing people - when eclcc does a compile and tries to retrieve an archive that is not there from the generated wu dll.